webapp/crashstats/crashstats/static/crashstats/css/components/message.less

```diff
@@ -24,9 +24,9 @@ div#message {
 }
 .message.warning {
     background-color: @pink;
-    border: 1px solid @jsonview-bool;
+    border: 1px solid @orange;
```

Comment:
I got rid of `jsonview-bool` when I updated the CSS for the jsonview. It's weird that the messages system was using that color. It was kind of a lighter tangerine color, so I replaced it with orange.